mockwebserver/src/main/kotlin/okhttp3/mockwebserver/MockResponse.kt

```diff
@@ -347,6 +349,10 @@ class MockResponse : Cloneable {
 
   override fun toString() = status
 
+  fun addInformationalResponseInternal(informationalResponse: MockResponse): MockResponse = apply {
```

Comment:
This is not great, but seems better to have some limited support without the bigger change of backporting.

cc @swankjesse